src/OpalCompiler-Core/IRTranslator.class.st

```diff
@@ -92,7 +92,7 @@ IRTranslator >> pushOuterVectors: scope [
 	| scopesWithVector sc |
 	scopesWithVector := OrderedCollection new.
 	sc := scope.
-	[ sc outerScope isNil ] whileFalse: [ 
+	[ sc outerScope isInstanceScope ] whileFalse: [ 
```

Comment:
Maybe better would be to ask a scope to pushTempVector. Then recursively it will travers all scopes up to OCInstanceScope